### PR TITLE
TextArea wrapper fix

### PR DESCRIPTION
**Pull request by victorgcramos**
This fixes the error message viewing for TextArea when it is wrapped by another parent component, to prevent the error space below the input, which was invisible, but still displayed on the screen.

### CODE REVIEW

File: src/components/TextArea/TextArea.jsx [hunk 1]
```diff
@@ -10,16 +10,23 @@ const TextArea = ({
...
   ...props
 }) => {
-  return (
-    <div className={classNames(styles.textAreaWrapper, wrapperClassNames)}>
+  return !wrapped ? (
+    <div
+      className={classNames(
+        styles.textAreaWrapper,
+        wrapperClassNames,
+        wrapped && styles.fullHeight
```
Comment (amassarwi) [inline src/components/TextArea/TextArea.jsx:21]: Actually I have a second thought here, I don't think this is needed also, I had something similar with the proposal filed input components, in some places I needed to override nested level of pi-ui's components, and the proper way to achieve that would be on `pigui` use something like:
`wrapperClassNames > div { full height or any styling rule you would like to override }` and passing `wrapperClassNames` should to the trick!

Comment (amassarwi) [reply src/components/TextArea/TextArea.jsx:21]: see: https://github.com/decred/politeiagui/blob/master/src/components/ProposalForm/ProposalForm.module.css#L27-L29

```
.typeSelectWrapper > div {
  height: 100%;
}
```

Comment (victorgcramos) [reply src/components/TextArea/TextArea.jsx:21]: Good observation!

File: src/components/TextArea/TextArea.jsx [hunk 2]
```diff
@@ -35,10 +36,26 @@ const TextArea = ({
         )}
       />
       <p
-        className={classNames(styles.errorMsg, error && styles.errorMsgActive)}>
+        className={classNames(
+          styles.errorMsg,
+          error && styles.errorMsgActive,
+          wrapped && styles.noErrorMsg
```
Comment (amassarwi) [inline src/components/TextArea/TextArea.jsx:42]: No need for `noErrorMsg` class here, see my other comment.

File: src/components/TextArea/TextArea.jsx [hunk 3]
```diff
@@ -35,10 +36,26 @@ const TextArea = ({
...
+        )}>
         {error}
```
Comment (amassarwi) [inline src/components/TextArea/TextArea.jsx:44]: You can get red of `noErrorMsg` and use the following:
```suggestion
        {!wrapped && error}
```

**Pull request merged**